osbenchmark/worker_coordinator/runner.py

```diff
@@ -61,8 +61,9 @@ def register_default_runners():
     register_runner(workload.OperationType.SubmitAsyncSearch, SubmitAsyncSearch(), async_runner=True)
     register_runner(workload.OperationType.GetAsyncSearch, Retry(GetAsyncSearch(), retry_until_success=True), async_runner=True)
     register_runner(workload.OperationType.DeleteAsyncSearch, DeleteAsyncSearch(), async_runner=True)
-    register_runner(workload.OperationType.OpenPointInTime, OpenPointInTime(), async_runner=True)
-    register_runner(workload.OperationType.ClosePointInTime, ClosePointInTime(), async_runner=True)
```

Comment:
Can we confirm none of the [workloads here](https://github.com/opensearch-project/opensearch-benchmark-workloads) do not use these operations

Reply:
Yes, I have checked and verified that these operations are not called in the workloads repo as of now